src/App.tsx

```diff
@@ -29,18 +33,20 @@ export function App() {
 		scrollRestoration: true,
 		context: {
 			queryClient,
+			authentication: authenticationContextValue,
 		},
 	});
 
 	return (
 		<>
-			<QueryClientProvider client={queryClient}>
+			<AuthenticationContext value={authenticationContextValue}>
```

Comment:
I pushed the QueryClientProvider out to the `main.tsx` so that we can utilize the api client in the `useRootAuthenticationContextValue()`.